modules/jaeger-config-storage.adoc

```diff
@@ -38,71 +27,43 @@ If you already have installed Elasticsearch as part of OpenShift logging, the Ja
 |`memory` or `elasticsearch`.
 Memory storage is only appropriate for development, testing, demonstrations, and proof of concept environments as the data does not persist if the pod is shut down. For production environments Jaeger supports Elasticsearch for persistent storage.
 |`memory`
-
 |===
 
+[id="jaeger-config-self-provisioning-es_{context}"]
+== Auto-provisioning an Elasticsearch instance
+
+When the `storage:type` is set to `elasticsearch` but there is no value set for `server-urls`, the Jaeger Operator uses the Elasticsearch Operator to create an Elasticsearch cluster based on the configuration provided in the `storage` section of the custom resource file.
+
+.Restrictions
+
+* There can be only one Elasticsearch per namespace.
+* You cannot share or reuse a {ProductName} logging Elasticsearch instance with Jaeger.  The Elasticsearch cluster is meant to be dedicated for a single Jaeger instance.
+
+[NOTE]
+====
+If you already have installed Elasticsearch as part of OpenShift logging, the Jaeger Operator can use the installed Elasticsearch Operator to provision storage.
+====
+
+The following configuration parameters are for a _self-provisioned_ Elasticsearch instance, that is an instance created by the Jaeger Operator using the Elasticsearch Operator.  You specify configuration options for self-provisioned Elasticsearch under `spec:storage:elasticsearch`  in your configuration file.
 
 .Elasticsearch configuration parameters
 [options="header"]
 [cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
-
 4+|General Elasticsearch configuration settings
 |elasticsearch:
   server-urls:
```

Comment:
This should be removed - the `server-urls` option is only specified for self provisioned ES clusters and does not belong under the `storage.elasticsearch` node, but under `storage.options.es`,